storage_service/locations/views.py

```diff
@@ -43,8 +43,13 @@ def get_delete_context_dict(request, model, object_uuid, default_cancel='/'):
 ########################## FILES ##########################
 
 def package_list(request):
-    packages = Package.objects.all()
-    return render(request, 'locations/package_list.html', locals())
+    context = {'packages': Package.objects.all()}
+    return render(request, 'locations/package_list.html', context)
+
+def package_fixity(request, package_uuid):
+    log_entries = FixityLog.objects.filter(package__uuid=package_uuid).order_by('-datetime_reported')
```

Comment:
Tip: You can filter by the value in a foreign key (instead of the object, which requires a join) by using `_id` https://docs.djangoproject.com/en/1.8/topics/db/queries/#field-lookups  So this could be `FixityLog.objects.filter(package_id=package_uuid).order_by('-datetime_reported')`